.gitignore

```diff
@@ -1,83 +0,0 @@
-# =============== #
```

Comment:
Did we mean to delete the .gitignore?

Reply:
i do not recall deleting it.... i can add it back

Reply:
added back